src/en/glossary.md

```diff
@@ -1,26 +1,23 @@
 # Glossary of terms
 
-**Bootstrap** - Initialize a Juju Environment so that Services may be deployed.
+**Bootstrap** - Initialize a Juju environment so that services may be deployed.
```

Comment:
I would also be fine using lower case for everything and not treating them as special unless they are being used in a particularly special way, which I haven't noticed so far